package.json

```diff
@@ -19,53 +19,43 @@
     "RESTful"
   ],
   "author": "SCODE-API",
-  "license": "ISC",
+  "license": "Apache-2.0",
```

Comment:
🤔  why the license change?

Reply:
I wondered the same...

Reply:
This actually isn't the license included with the project! The `LICENSE` file contains the Apache 2.0 license. I could change that one instead, but I'm thinking those should match up